features/step_definitions/admin_steps.rb

```diff
@@ -101,6 +101,7 @@
 Given /^I have posted an admin post$/ do
   step(%{I am logged in as a "communications" admin})
   step("I make an admin post")
+  step("I am on the homepage")
```

Comment:
Without this, the JS-enabled tests fail as we land on example.com after making the admin post. I don't know why that is...is it related to [this default URL value](https://github.com/otwcode/otwarchive/blob/77a69888b51a0403ffbf5da0b1f3da1df56e2722/config/environments/test.rb#L48)? Please suggest a better fix if you know one!

Reply:
It's been a while, but there is a better fix now:  
In `AdminPostsController#create` change the `redirect_to(@admin_post)` to `redirect_to admin_post_path(@admin_post)`.

Explanation:
The controller calls `redirect_to` on a model object (instead of a path). When you call `redirect_to` on a model, Rails automatically tries to convert it to a full URL, instead of a path, and when the host is set to `example.com`, that means that it tries to redirect to `example.com`.